src/App.js

```diff
@@ -1,10 +1,10 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.css';
 
 import Board from './components/Board';
 
-const player_1 = 'X';
-const player_2 = 'O';
+const player1 = 'x';
+const player2 = 'o';
```

Comment:
We gave bad guidance here. This should have been
```js
const PLAYER_1 = 'x';
const PLAYER_2 = 'o';
```
since the tests were all written for lower case x and o. But it looks like you took care of it.